symbolizer-ffi/src/blazesym.h

```diff
@@ -0,0 +1,895 @@
+// BSD 3-Clause License
+// Synchronized from blazesym repository
```

Comment:
How will we make sure this stays in sync?

Reply:
I could do a clone / copy in a build script, but I don't like adding git steps to the build process.
Maybe if this is a conditional build step :thinking: